src/widget/groupwidget.cpp

```diff
@@ -145,7 +147,13 @@ void GroupWidget::onUserListChanged()
 {
     Group* g = GroupList::findGroup(groupId);
     if (g)
-        statusMessageLabel->setText(tr("%1 users in chat").arg(g->getPeersCount()));
+    {
+        int peersCount = g->getPeersCount();
+        if (peersCount == 1)
+            statusMessageLabel->setText(tr("1 user in chat"));
+        else
+            statusMessageLabel->setText(tr("%1 users in chat").arg(peersCount));
+    }
     else
         statusMessageLabel->setText(tr("0 users in chat"));
```

Comment:
Is it possible to have 0 users in a group chat? If you are in there is 1 user, you, and if you aren't then it doesn't even exist